spyder/utils/icon_manager.py

```diff
@@ -161,6 +163,8 @@
     'gotoline':                [('fa.sort-numeric-asc',), {'color': MAIN_FG_COLOR}],
     'error':                   [('fa.times-circle',), {'color': 'darkred'}],
     'warning':                 [('fa.warning',), {'color': 'orange'}],
+    'information':             [('fa.info-circle',), {'color': '#3775a9'}],
+    'hint':                    [('fa.lightbulb-o',), {'color': 'yellow'}],
```

Comment:
>Instead of using the information sign for hints, please use mdi-lightbulb to follow VSCode lead regarding that symbol.

@ccordoba12 this is already the case, but none of those messages are hints, they are information. I am not sure how to even get a hint

Reply:
Ok, don't worry, I thought those were hints. Let's go for using a shade of green for now.